### ISSUE TITLE
4.1.1 Parsing: The WCAG 2.1 version of the SC is not quoted, yet a word substitution is provided for it

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#424]
### Currently

Here is the simplified structure currently appearing:

- h5. 4.1.1 Parsing (Obsolete and removed)
  - (quote of the WCAG 2.2 version of SC 4.1.1)
  - h6. Applying SC 4.1.1 Parsing to Non-Web Documents and Software
    - WCAG 2.2 Guidance:
      - (Note for the WCAG 2.2 version of SC 4.1.1)
    - WCAG 2.0 and 2.1 Guidance:
      - This applies directly as written, and as described in Intent from Understanding Success Criterion 4.1.1, replacing...
      - With this substitution, it would read:...

Main issue: under "WCAG 2.0 and 2.1 Guidance" it says "This applies directly as written", but there's nothing for "this" to refer to. The WCAG 2.1 version of SC 4.1.1 is not present in the page.

Issue: [4.1.1 in WCAG 2.0](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#ensure-compat-parses) has one note. [4.1.1 in WCAG 2.1](https://w3c.github.io/wcag2ict/#parsing) has two notes, which are different from the one in WCAG 2.0. None of these notes appear under "WCAG 2.0 and 2.1 Guidance" in WCAG2ICT, nor are they mentioned in the description of word substitution.

Issue: Everything is under one heading, "4.1.1 Parsing (Obsolete and removed)" which refers only to the WCAG 2.2 version of SC 4.1.1. This heading does not accurately describe the section of content below it. It's confusing (and a failure of WCAG SC 2.4.6 Headings and Labels).

Issue: "WCAG 2.2 Guidance" and "WCAG 2.0 and 2.1 Guidance" function visually as headings but are not marked up as headings.

Issue: Under "WCAG 2.2 Guidance" there is an added "Note 1". Under "WCAG 2.0 and 2.1 Guidance" the note numbering does not restart, but appears as "Note 2... Note 3... Note 4". One could argue this is correct as-is, but to me it feels slightly off.

### Proposed

Restructure the section. If it's doesn't blow up the table of contents and the scripts too badly, something like this:

- h5. 4.1.1 Parsing (Obsolete and removed) — WCAG 2.2
  - (quote of the WCAG 2.2 version of SC 4.1.1)
  - h6. Applying SC 4.1.1 Parsing (WCAG 2.2) to Non-Web Documents and Software
    - (Note for the WCAG 2.2 version of SC 4.1.1)
- h5. 4.1.1 Parsing — WCAG 2.0 and 2.1
  - (quote of the WCAG 2.1 version of SC 4.1.1)
  - h6. Applying SC 4.1.1 Parsing (WCAG 2.0 and 2.1) to Non-Web Documents and Software
      - This applies directly as written, and as described in Intent from Understanding Success Criterion 4.1.1, replacing...
      - With this substitution, it would read:...

Include a quote of the WCAG 2.1 version of SC 4.1.1 at the appropriate spot in the structure.

In the description of word substitution for WCAG 2.0 and 2.1, say that the Notes for WCAG 2.0 and WCAG 2.1 have been removed. See #434 for an example of how this can be phrased.

**Comment by maryjom** [comment]
@daniel-montalvo We'll have to work together on this, as some special scripting would be required to fix this as described. It would require a general heading for 4.1.1 Parsing with two sub-headings - one to include the WCAG 2.2 version of 4.1.1 Parsing and one to include the WCAG 2.1 version of 4.1.1 Parsing. We could do this the quick method by copying over the WCAG 2.1 original text into the markdown rather than an include. If we do that, we'll have to verify that the scripting to include WCAG 2.2 version, the note numbering, and the automated heading replacement will work when it's in a heading that's one level deeper.

**Comment by daniel-montalvo** [comment]
Thanks @maryjom  and @mitchellevan 

I plan to take a look at this early next week. I'll try first adding a special treatment for WCAG 2.1's 4.1.1 via scripting and if it does not work the nwe can try adding it manually to the markdown, but it would be od.
